annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/WebAnnotationSearchRest.java

```diff
@@ -112,8 +112,8 @@ private ResponseEntity<String> searchAnnotation(String wskey, String queryString
 	    }
 
 	    // ** serialize page
-	    AnnotationPageSerializer serializer = new AnnotationPageSerializer(annotationPage);
-	    String jsonLd = serializer.serialize(searchProfile);
+	    annotationPageSerializer.setProtocolPage(annotationPage);
```

Comment:
serializer is statefull, therefore a new object must be created for each request